File: KnowSongJPA/src/main/java/com/skilldistillery/knowsong/entities/Playlist.java
package com.skilldistillery.knowsong.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Playlist {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	
	
	@Column(name = "spotify_id")
	private String spotifyId;
	
	private String name;
	
	private String description;
}

File: ngKnowSong/src/app/spotifyJSON/models/playlist.ts
import { Track } from './track';

export class Playlist {
  public id: string;  //spotify id
  public description: string; //public description of playlist
  public name: string; //name of playlist
  public tracks: Track[];
  constructor(id?:string, name?:string, description?:string, tracks?:Track[] ){
    this.id = id;
    this.description = description;
    this.name = name;
    this.tracks = tracks
  }

}
